spec/models/export/activity_actuals_columns_spec.rb

```diff
@@ -128,6 +128,21 @@
         expect(value_for_activity).to eq BigDecimal("300")
         expect(last_column_data.count).to eq 5
       end
+
+      context "if, unexpectedly, there is no value for that report quarter" do
+        let(:activity) { create(:project_activity) }
+        subject {
+          described_class.new(activities: [activity], include_breakdown: breakdown, report: report)
+        }
+
+        it "returns zero and not nil" do
+          last_column_data = subject.rows_for_last_financial_quarter
+          value_for_activity = last_column_data.fetch(activity.id)
+
+          expect(value_for_activity).to eq 0
```

Comment:
Looks like there's a [be_zero matcher](https://relishapp.com/rspec/rspec-expectations/v/2-2/docs/matchers/predicate-matchers), but I don't have a particular preference for that

Reply:
I've taken this from the existing example test in the PR above, but good to know about this matcher!